game/scripts/npc/items/force_weapons/item_pull_staff.txt

```diff
@@ -106,7 +106,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_health_regen"                              "12 18 27 40"
+        "bonus_health_regen"                              "10 15 25 40"
```

Comment:
should probably match forcestaff levels "10 20 40 80"

Reply:
Force staff has twice the regen because it did before, too. It's actually a relative buff. I wasn't sure if pull staff was intentionally weaker since the effect is stronger